pycbc/workflow/grb_utils.py

```diff
@@ -602,30 +603,24 @@ def make_pygrb_injs_tables(workflow, out_dir,  # exclude=None, require=None,
     # Initialize job node
     grb_name = workflow.cp.get('workflow', 'trigger-name')
     extra_tags = ['GRB'+grb_name]
-    # TODO: why is inj_set repeated twice in output files?
-    if inj_set is not None:
-        extra_tags.append(inj_set)
     node = PlotExecutable(workflow.cp, exec_name,
                           ifos=workflow.ifos, out_dir=out_dir,
                           tags=tags+extra_tags).create_node()
+    # Pass the bank-file
+    node.add_input_opt('--bank-file', resolve_url_to_file(bank_file))
```

Comment:
Same comment here with `bank_file`, pass `File` objects around, not URLs. There are potential failure modes with file handling here if `resolve_url_to_file` doesn't realise it is the same file every time.